src/UI/Art/SpriteArtist/InfoView.java

```diff
@@ -53,81 +57,111 @@ public void render(Graphics g)
-            drawingWidth, paneVSize, thisCO);
-        myG.drawImage(headersImage, 2*paneOuterBuffer, 2*paneOuterBuffer, null);
-        break;
-      case GAME_STATUS:
-        if( null != thisCO && thisCO != statsCO )
-        {
-          statsCO = thisCO;
-          COStateInfo coStatus = new COStateInfo(getDrawableMap(myControl.getGame()), thisCO);
-          BufferedImage[] statusHalves = new BufferedImage[2];
-          // Draw assuming we don't have to wrap the text
-          statusHalves[0] = SpriteUIUtils.drawProse(coStatus.getFullStatusLabels());
-          statusHalves[1] = SpriteUIUtils.drawProse(coStatus.getFullStatusValues());
-          statsImage = SpriteUIUtils.joinBufferedImages(statusHalves, 3);
-        }
-        if( null != statsImage )
-          myG.drawImage(statsImage, 3 * paneOuterBuffer, 3 * paneOuterBuffer, null);
-        break;
-      case BASIC:
-        BufferedImage infoText = SpriteUIUtils.drawProseToWidth(page.info, drawingWidth);
-        myG.drawImage(infoText,3*paneOuterBuffer, 3*paneOuterBuffer, null);
-        break;
-    }
-    // Destroy stale state so we don't get into weird situations
-    if( page.pageType != PageType.GAME_STATUS )
-      statsCO = null;
+    myG.fillRect(2*PANE_OUTER_BUFFER, 2*PANE_OUTER_BUFFER, paneHSize                      , paneVSize                      );
+
+    int drawingWidth = paneHSize - PANE_OUTER_BUFFER;
+    if( pages != prevPages )
+      pageImage = renderPage(pages, drawingWidth);
+    prevPages = pages;
+
+    final int shiftDown = myControl.getShiftDown() * 2;
+    final int possibleShift = Math.max(0, pageImage.getHeight() - paneVSize);
+    int pixelShift = 0;
+    if( possibleShift > 0 )
+      pixelShift = (possibleShift + shiftDown % possibleShift) % possibleShift;
+
+    final int contentWidth  = Math.min(Math.abs(paneHSize - PANE_OUTER_BUFFER), pageImage.getWidth());
+    final int contentHeight = Math.min(Math.abs(paneVSize - PANE_OUTER_BUFFER), pageImage.getHeight());
+    // Draw the cropped page into our pane
+    myG.drawImage(pageImage.getSubimage(0, pixelShift, contentWidth, contentHeight), 3 * PANE_OUTER_BUFFER, 2 * PANE_OUTER_BUFFER, null);
 
     // Finally, draw our rendered image onto the window.
     g.drawImage(image, 0, 0, imageWidth*drawScale, imageHeight*drawScale, null);
   }
 
-  
+  private BufferedImage renderPage(ArrayList<InfoPage> pages, int drawingWidth)
+  {
+    ArrayList<BufferedImage> pageImages = new ArrayList<BufferedImage>();
+    for( InfoPage p : pages )
+    {
+      switch (p.pageType)
+      {
+        case CO_HEADERS:
+        case GAME_STATUS: // Fall-through
+          Commander[] coList = myControl.getGame().commanders;
+          MapPerspective drawableMap = getDrawableMap(myControl.getGame());
+
+          for( Commander thisCO : coList )
+          {
+            BufferedImage statusHeader =
+                SpriteLibrary.createTransparentSprite(drawingWidth, CommanderOverlayArtist.OVERLAY_HEIGHT + 5);
```

Comment:
Is it +5 to make room for the power bar?

Reply:
Yes; that's copied from the CO overlay artist, and probably should be a constant?